packages/blockchain-extension/tutorials/developer-tutorials/add-environment.md

```diff
@@ -48,37 +48,37 @@ The identities that were exported need to be imported into VS Code. First we wil
 
 2. You will then be asked to enter a name for the wallet use `ibp-wallet`.
 
-3. You will then be asked to provide a name for the identity use `Org1 Admin`.
+3. You will then be asked to provide the MSPID use `org1msp` if you followed the tutorial.
 
-4. You will then be asked to provide the MSPID use `org1msp` if you followed the tutorial.
+4. You will then be asked for the method for adding an identity choose `Provide JSON identity file from IBM Blockchain Platform`.
 
-5. You will then be asked for the method for adding an identity chose `Provide JSON identity file from IBM Blockchain Platform`.
+5. Browse to the location of the previously exported `Org1 Admin` file and select it.
 
-6. Browse to the location of the previously exported `Org1 Admin` file and select it.
+6. Alternatively, If the JSON identity file does not contain the name property, you will be asked to enter a name for the identity.
```

Comment:
We're saying `Alternatively`, but not what this is an alternative to. Perhaps something like this:
6. If the JSON identity file does not contain the name property, you will be asked to enter a name for the identity, otherwise this step is skipped.